src/main/java/org/fundacionjala/coding/ariel/CesarVigenereEncryption/VigenereEncryption.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.coding.ariel.cesarvigenereencryption;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "VigenereEncryption.java" should be located in "org/fundacionjala/coding/ariel/cesarvigenereencryption" directory, not in "/home/travis/build/AT-06/coding/src/main/java/org/fundacionjala/coding/ariel/CesarVigenereEncryption". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)